python/src/opendp/mod.py

```diff
@@ -798,6 +801,50 @@ def __hash__(self) -> int:
     def __iter__(self):
         raise ValueError("Metric does not support iteration")  # pragma: no cover
 
+class MultiDistance(Metric):
+    '''`MultiDistance` is a higher-order metric with multiple distance types for grouped data.'''
+
+    _type_ = AnyMetric
+    
+    @property
+    def inner_metric(self) -> Metric | None:
+        '''Bounds of the domain, if they exist'''
+        from opendp.metrics import _multi_distance_get_inner_metric
+        return _multi_distance_get_inner_metric(self)
+    
+    def __del__(self):
+        # this is to avoid a double-free, as the Domain.__del__ will, suprisingly, still be called
+        pass
+
+class SymmetricIdDistance(Metric):
+    '''`SymmetricIdDistance` is a metric for measuring the distance between the identifiers of two datasets.'''
```

Comment:
```suggestion
    '''``SymmetricIdDistance`` is a metric for measuring the distance between the identifiers of two datasets.'''
```

Reply:
It's in the name, but probably want a sentence in the docstring to distinguish this from `ChangeOneIdDistance`.